docker/README.md

```diff
@@ -0,0 +1,37 @@
+Sawtooth Lake - Docker
```

Comment:
It's substantially included under the developers guide in the docker documentation updates I have a PR for. https://github.com/hyperledger/sawtooth-core/pull/283